Google.Api.Generator.Rest/Models/MethodModel.cs

```diff
@@ -219,43 +219,61 @@ private static ClassDeclarationSyntax AddMediaDownloadMethods(PropertyDeclaratio
             var cancellationTokenWithDefault = Parameter(ctx.Type<CancellationToken>(), "cancellationToken", DefaultExpression(ctx.Type<CancellationToken>()));
             var range = Parameter(ctx.Type<RangeHeaderValue>(), "range");
 
-             var syncDownloadWithStatus = Method(Modifier.Public | Modifier.Virtual, ctx.Type<IDownloadProgress>(), "DownloadWithStatus")(stream)
-                .WithBlockBody(Return(mediaDownloader.Call("Download")(ThisQualifiedCall("GenerateRequestUri")(), stream)))
+            var downloadRemarks = XmlDoc.Remarks("This method uses the ", mediaDownloader, " property to perform the download. " +
```

Comment:
```suggestion
            var downloadRemarks = XmlDoc.Remarks("This method uses the ", MediaDownloader, " property to perform the download. " +
```

Reply:
`mediaDownloader` here is a variable representing the property declaration - the `XmlDoc.Remarks` method automatically turns that into a `<see>` link.

Reply:
Ah that's great!